tensorflow/python/util/compat_test.py

```diff
@@ -0,0 +1,77 @@
+# Copyright 2017 The TensorFlow Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+"""Unit tests for compat."""
+
+# pylint: disable=unused-import
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+try:
+  from pathlib import Path
+except ImportError:
+  Path = None
+
+try:
+  from collections.abc import Iterable as _Iterable
+except ImportError:
+  from collections import Iterable as _Iterable
+
+import inspect
+
+from tensorflow.python.platform import test
+from tensorflow.python.platform import tf_logging as logging
+from tensorflow.python.util import compat
+from tensorflow.python.ops import array_ops
+
+class CompatPathToStrTest(test.TestCase):
+  """Tests for compat path to string utilities"""
+
+  def test_transforms_pathlib_path(self):
+    if Path is None:
+      return # Skip test if pathlib not available.
```

Comment:
Use `self.skipTest("Test requires Python 3.6 or greater.")` here and below.

Reply:
Thanks, I've replaced return statements in both tests targeting pathlib